### PR TITLE
Testing improvements to cluster and pool stats.

### PR DESCRIPTION
**Pull request by neurodrone**
* Adds testing for extraneous and invalid cases.

cc: @mdlayher @cagedmantis

### CODE REVIEW

File: collectors/cluster_usage.go [hunk 1]
```diff
@@ -106,29 +106,32 @@ func (c *ClusterUsageCollector) collect() error {
 		return err
 	}
 
-	tot, err := stats.Stats.TotalBytes.Float64()
+	var totBytes, usedBytes, availBytes, totObjects float64
+
+	totBytes, err = stats.Stats.TotalBytes.Float64()
 	if err != nil {
-		return err
+		log.Println("[ERROR] cannot extract total bytes:", err)
```
Comment (mdlayher) [inline collectors/cluster_usage.go]: `return` after error?

Also, might be worth making `infof()` and `errorf()` functions to save you the hassle of adding this prefix and such.

Comment (neurodrone) [inline collectors/cluster_usage.go]: I have changed the logic to show blips to separate stats-related failures with network-related ones. This case should almost never be triggered and there should be a strong visible sign on our graph if it ever does.

Easy enough to add prefixes for now but good to have that option once we start using them extensively.

**Comment by neurodrone** [comment]
@mdlayher Let me know if you have any concerns around this. :)

**Comment by mdlayher** [comment]
:+1:

**Pull request merged**